<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastFmConfigWidget</class>
 <widget class="QWidget" name="LastFmConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>416</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox3">
     <property name="title">
      <string>Last.fm Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelUsername">
        <property name="text">
         <string>&amp;Username:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_ScrobblerUsername</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPassword">
        <property name="text">
         <string>&amp;Password:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_ScrobblerPassword</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="kcfg_ScrobblerPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="kcfg_ScrobblerUsername"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="kActiveLabel1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&lt;a href=&quot;http://www.last.fm:80/signup.php&quot;&gt;Sign up to Last.fm&lt;/a&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="testLogin">
        <property name="text">
         <string>&amp;Test Login</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Last.fm Services</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="kcfg_SubmitPlayedSongs">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Submit tracks</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_RetrieveSimilarArtists">
        <property name="text">
         <string>&amp;Retrieve similar artists</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_ScrobbleComposer">
        <property name="text">
         <string>&amp;Use composer data if available in Last.fm as artist</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_UseFancyRatingTags">
        <property name="toolTip">
         <string>Use Last.fm tags like &lt;b&gt;7 of 10 stars&lt;/b&gt; to represent your Amarok ratings during statistics synchronization</string>
        </property>
        <property name="text">
         <string>Use fancy tags to represent ratings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_AnnounceCorrections">
        <property name="toolTip">
         <string>&lt;html&gt;When scrobbling tracks to Last.fm, it autocorrects common errors in track tags like artist, album and title. Check this to be notified when this happens and what the autocorrection was&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Announce correction suggestions for track tags</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="kcfg_FilterByLabel">
          <property name="toolTip">
           <string>&lt;html&gt;Check if you want certain tracks neither to be scrobbled nor to be updated to Last.fm now playing&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Do not scrobble tracks with label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="kcfg_FilteredLabel">
          <property name="toolTip">
           <string>&lt;html&gt;Select a preferred label (or write a new one). Tracks with this label will not be scrobbled&lt;/html&gt;</string>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>kcfg_ScrobblerUsername</tabstop>
  <tabstop>kcfg_ScrobblerPassword</tabstop>
  <tabstop>testLogin</tabstop>
  <tabstop>kcfg_SubmitPlayedSongs</tabstop>
  <tabstop>kcfg_RetrieveSimilarArtists</tabstop>
 </tabstops>
 <includes>
  <include location="local">qlineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
